.circleci/config.yml

```diff
@@ -114,10 +114,26 @@ jobs:
     steps:
       - checkout
 
+      - run:
```

Comment:
Why?

Reply:
node is required react-native-cli for generating source maps